lib/ansible/utils/module_docs_fragments/openstack.py

```diff
@@ -69,12 +60,35 @@ class ModuleDocFragment(object):
       - How long should ansible wait for the requested resource.
     required: false
     default: 180
+  api_timeout:
+    description:
+      - How long should the socket layer wait before timing out for API calls.
+        If this is omitted, nothing will be passed to the requests library.
+    required: false
+  verify:
```

Comment:
is this the same as "validate_certs" in other modules? if so I would prefer to standardize the naming to that